CHANGELOG/CHANGELOG-0.1.md

```diff
@@ -0,0 +1,31 @@
+# Changelog 0.1
+
+## 0.1.0 - 2022-06-16
+
+This is the first Theia release and it includes all Antrea flow visibility capabilities released as of 1.6.
+
+
+### Added
+
+- Helm support for Theia installation ([#15](https://github.com/antrea-io/theia/pull/15) [@yanjunz97]) ([#31](https://github.com/antrea-io/theia/pull/31) [@yanjunz97]) ([#34](https://github.com/antrea-io/theia/pull/34)  [@yanjunz97])
+  * Provide Helm charts for Theia components (Spark operator, Clickhouse, and Grafana)
+  * Allow for deploying Clickhouse on a persistent volume
+- Grafana dashboard: add a panel for visualizing denied connections ([#12](https://github.com/antrea-io/theia/pull/12) [@heanlan])
+  * Introoduce chord diagram for visualizing connections denied by network policies
+  * Reorganized Network policy dashboard to visualize both allowed and denied traffic
+- Policy recommendation via Spark job ([#16](https://github.com/antrea-io/theia/pull/16) [@dreamtalen])
+  * Add a spark job to perform network policy recommendation based on flow data pushed by Flow Aggregator
+- Add Kind e2e test for Theia ([#14](https://github.com/antrea-io/theia/pull/14) [@wsquan171])
+- Add Jenkins e2e jobs ([#20]https://github.com/antrea-io/theia/pull/20) [@hangyan]
+
+
+### Fixed
+
+- A redundant check in e2e tests for Flow aggreator has been removed ([#22](https://github.com/antrea-io/theia/pull/22) [@dreamtalen]
+- Add a limit to the quantify of data rendered by Grafana dashboard to avoid bad performance, fix indirect cycle handing in Sankey panel ([#42](https://github.com/antrea-io/theia/pull/42) [@heanlan])
```

Comment:
"fix indirect cycle handing in Sankey panel" -> "add indirect cycle handing in Sankey panel"

Shall we move this item from `Fixed` to `Changed` ?